amazon-sqs/5.11/modules/ROOT/pages/amazon-sqs-connector-troubleshooting.adoc

```diff
@@ -8,6 +8,14 @@ include::connectors::partial$verbose-exception-logs.adoc[]
 
 include::connectors::partial$verbose-logs-org.adoc[]
 
+== Troubleshoot AWS STS Regionalized Endpoints
+
+Since the upgrade of AWS SDK from v1 to v2, the endpoint that the STS client of the AssumeRoleConnectionProvider uses to get session tokens now is Regionalized instead of Global. Refer to the https://docs.aws.amazon.com/sdkref/latest/guide/feature-sts-regionalized-endpoints.html[AWS STS Regionalized endpoints] documentation for details.
```

Comment:
In AWS SDK v2, the endpoint that the STS client of the `AssumeRoleConnectionProvider` operation uses to get session tokens is **Regionalized** instead of **Global**. See the https://docs.aws.amazon.com/sdkref/latest/guide/feature-sts-regionalized-endpoints.html[AWS STS Regionalized endpoints] documentation for details.